docs/UsersGuide.asciidoc

```diff
@@ -193,28 +193,36 @@ by different means:
 
 
 === Variable expansion
+Any job setting can refer to another variable using this syntax: `%NAME%`. When
+the test job is created, the string will be substituted with the value of the
+specified variable at that time.
 
-Any variable defined in Test Suite, Machine, Product or Job Template table can
-refer to another variable using this syntax: `%NAME%`. When the test job is created,
-the string will be substituted with the value of the specified variable at that time.
+The variable expansion applies to job settings defined in test suites, machines,
+products and job templates. It also applies to job settings specified when
+creating a single set of jobs and to variables specified in the worker config.
```

Comment:
```suggestion
creating a single set of jobs and to variables specified in the worker config at the time of job exectution.
```

Reply:
not sure if this applies, if the above is added.

Reply:
I wanted to keep it short. Our documentation is long enough and the addition seems redundant.